syntax = "proto3";

package rti;

message RequestLogin
	{                                
        enum SysInfraType {
    SYSINFRATYPE_UNSPECIFIED = 0;
	                   TICKER_PLANT      = 1;
			   ORDER_PLANT       = 2;
			   HISTORY_PLANT     = 3;
			   PNL_PLANT         = 4;
			   REPOSITORY_PLANT  = 5;
	                  } 
     
	int32        template_id         = 154467;
	string       template_version    = 153634;
	repeated string       user_msg            = 132760;

	string       user                = 131003;
	string       password            = 130004;
	string       app_name            = 130002;
	string       app_version         = 131803;
	string       system_name         = 153628;
	SysInfraType infra_type          = 153621;
	repeated string       mac_addr            = 144108;
	string       os_version          = 144021;
	string       os_platform         = 144020;
	bool         aggregated_quotes   = 153644;  // applicable only for TICKER_PLANT infra_type
	}
